shared/common-adapters/usernames/index.tsx

```diff
@@ -86,9 +86,9 @@ const UsernameText = (props: Props) => {
         const _onUsernameClicked = props.onUsernameClicked
         const isNegative = backgroundModeIsNegative(props.backgroundMode || null)
         const renderText = (onLongPress?: () => void) => (
-          <Text type={props.type} key={u.username}>
+          <Text type="Body" key={u.username}>
```

Comment:
Curiosity question: is this `"Body"` change necessary?

Reply:
yes, else this is BodyBoldLink and the entire hierarchy gets the hover behavior